bin/travis/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function bloop_version {
+  cat build.sbt | grep "val bloop" | sed 's|[^0-9.]||g'
+}
+
+wget -O bin/coursier https://git.io/coursier-cli && chmod +x bin/coursier \
+&& bin/coursier launch ch.epfl.scala:bloopgun-core_2.12:$(bloop_version) -- server &>bloop.log &
```

Comment:
You don't need to run `bloop server` here, you can instead run:

`bin/coursier launch ch.epfl.scala:bloopgun-core_2.12:$(bloop_version) -- about`

That invocation will make sure of starting the server in the background if there isn't one running. You'll lose access to the logs but it's unclear why you want to have access to them here, the server logs do not contain any useful information and are mostly noise coming from the nailgun server.